src/TerminalObject/Basic/Flank.php

```diff
@@ -67,7 +67,7 @@ public function repeat($repeat)
      */
     public function result()
     {
-        $flank = str_repeat($this->char, $this->repeat);
+        $flank = str_repeat($this->char ?? '', $this->repeat ?? 0);
```

Comment:
Both of these properties are declared above, what scenario is this to handle?

Reply:
revert it